SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -1019,5 +1020,68 @@ private void GetScale()
                     SendMessageToPC(user, $"{targetName} has a scale of {shownScale}.");
                 });
         }
+
+        private void ShipStats()
+        {
+            _builder.Create("shipstats")
+                .Description("Provides a readout of a given ship's stats.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .AvailableToAllOnTestEnvironment()
+                .RequiresTarget()
+                .Action((user, target, location, args) =>
+                {
+                    if (GetIsDM(target) != true && GetIsDMPossessed(target) != true)
+                    {
+                        if (Space.GetShipStatus(target) != null)
+                        {
+                            var npcStats = Stat.GetNPCStats(target);
+                            var level = npcStats.Level;
+                            if (GetIsPC(target) == true)
+                            {
+                                var playerId = GetObjectUUID(target);
+                                var dbPlayer = DB.Get<Player>(playerId);
+                                level = dbPlayer.Skills[Service.SkillService.SkillType.Piloting].Rank;
+                            }
+                            var targetName = GetName(target);
+                            var targetStatus = Space.GetShipStatus(target);
+                            SendMessageToPC(user, $"{targetName} stats: \n" +
+                                $"Armor: {targetStatus.Hull} Max: {targetStatus.MaxHull} \n" +
+                                $"Shields: {targetStatus.Shield} Max: {targetStatus.MaxShield} \n" +
+                                $"Capacitor: {targetStatus.Capacitor} Max: {targetStatus.MaxCapacitor} \n" +
+                                $"Shield Regen: {targetStatus.ShieldRechargeRate} \n" +
+                                $"EM Damage Bonus: {targetStatus.EMDamage} \n" +
+                                $"Thermal Damage Bonus: {targetStatus.ThermalDamage} \n" +
+                                $"Explosive Damage Bonus: {targetStatus.ExplosiveDamage} \n" +
+                                $"Accuracy: {targetStatus.Accuracy} \n" +
+                                $"Evasion: {targetStatus.Evasion} \n" +
+                                $"Thermal Defense: {targetStatus.ThermalDefense} \n" +
+                                $"EM Defense: {targetStatus.EMDefense} \n" +
+                                $"Explosive Defense: {targetStatus.ExplosiveDefense} \n" +
+                                $"Skill Level: {level}");
+                        }
+                    }
+                });
+        }
+
+        private void RepairShip()
+        {
+            _builder.Create("repairship")
```

Comment:
Could you just bake this in to the existing /rez command?

Reply:
The advantage here is I figure DM's can apply damage manually to ships with this command as well, or to only repair a certain amount. At present there's no way for DM's to do either of those things with ships. I can bake it into rez instead if you'd prefer to not have another chat command.

Reply:
I can kind of see the value of restoring by a specific amount, but if you really think that will be used we can keep it. The only thing I'd ask is that if a number isn't provided, do a full recovery on the ship. Right now it'll do nothing unless a number is entered, and I know people are going to forget.